### PR TITLE
feat(frontend): 新增spider授权白名单页面

### PR DESCRIPTION
**Pull request by 3octaves**

### CODE REVIEW

**Comment by tencentblueking-adm** [comment]
[CLA assistant check](https://bk-cla.bktencent.com:443/TencentBlueKing/blueking-dbm?pullRequest=640) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://bk-cla.bktencent.com:443/TencentBlueKing/blueking-dbm?pullRequest=640) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://bk-cla.bktencent.com:443/check/TencentBlueKing/blueking-dbm?pullRequest=640) it.</sub>

File: dbm-ui/frontend/src/services/spider/permission.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+/**
+ * 查询账号规则列表返回结果
+ */
+export type PermissionRulesResult =  ListBase<Permission[]>
+
+/**
+ * 查询账号规则列表
+ */
+export const getPermissionList = (params: PermissionRulesParams): Promise<PermissionRulesResult> => http.get(`/apis/mysql/bizs/${params.bk_biz_id}/permission/account/list_account_rules/`, params);
```
Comment (jinquantianxia) [inline dbm-ui/frontend/src/services/spider/permission.ts:71]: 1. 返回类型放get那里，如:
`http.get<PermissionRulesResult>`

2. params.bk_biz_id可以优化成统一使用useGlobalBizs去获取，如：
`import { useGlobalBizs } from '@stores';
const { currentBizId } = useGlobalBizs();
`/apis/mysql/bizs/${currentBizId }/permission/account/list_account_rules/`
`

File: dbm-ui/frontend/src/views/spiderss-manage/permission/Index.vue [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+
+  const columns = [
+    {
+      label: t('账号名称'),
+      field: 'user',
+      width: 200,
+      minWidth: 200,
+      showOverflowTooltip: false,
+      render: ({ data }: { data: PermissionTableRow }) => (
+      <div class="permission__cell" onClick={ handleToggleExpand.bind(null, data) }>
```
Comment (jinquantianxia) [inline dbm-ui/frontend/src/views/spiderss-manage/permission/Index.vue:101]: 函数中没有用到this，也不需要改变this的指向，可以不必bind，可以优化成如下：
` render: ({ index, data }: { index: number, data: PermissionTableRow }) => (
<div class="permission__cell" onClick={ () => handleToggleExpand(index, data) }>`
其他类似的也可以自行优化下

File: dbm-ui/frontend/src/views/spiderss-manage/permission/Index.vue [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+  function handleSearch() {
+    nextTick(getList);
+  }
+
+  const handleToggleExpand = (data: PermissionTableRow) => {
+    // 长度小于等于 2 则没有展开收起功能
+    if (data.rules.length <= 1) return;
+    // eslint-disable-next-line no-param-reassign
+    data.isExpand = !data.isExpand;
+  };
```
Comment (jinquantianxia) [inline dbm-ui/frontend/src/views/spiderss-manage/permission/Index.vue:211]: 结合前面的comment，此处可以优化成
`  const handleToggleExpand = (index: number, data: PermissionTableRow) => {
    // 长度小于等于 2 则没有展开收起功能
    if (data.rules.length <= 1) return;
    state.data[index].isExpand = !data.isExpand;
  };`

File: dbm-ui/frontend/src/views/spiderss-manage/permission/common/const.ts [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+export const dbOperations = {
+  dml: ['select', 'insert', 'update', 'delete'],
+  ddl: ['create', 'alter', 'drop', 'index', 'execute'],
+  glob: ['replication client', 'replication slave', 'file'],
+};
+
+/**
+ * 密码策略
+ */
+export enum PASSWORD_POLICY {
```
Comment (jinquantianxia) [inline dbm-ui/frontend/src/views/spiderss-manage/permission/common/const.ts:26]: 考虑下如何适应国际化要求？

**Pull request closed without merging**